bridge/deploy/05_deploy_bridgeProxy.js

```diff
@@ -30,19 +31,21 @@ module.exports = async function (hre) { // HardhatRuntimeEnvironment
     methodCall.encodeABI()
   ];
 
-  const deployProxyResult = await deploy('BridgeProxy', {
+  const deployResult = await deploy('BridgeProxy', {
```

Comment:
Let the name deployProxyResult because we are deploying a proxy contract in here